api/integrations/github/github.py

```diff
@@ -25,6 +26,21 @@
 logger = logging.getLogger(__name__)
 
 
+def handle_installation_deleted(payload: dict[str, typing.Any]) -> None:
+    installation_id = payload.get("installation", {}).get("id")
+    try:
+        GithubConfiguration.objects.get(installation_id=installation_id).delete()
+    except GithubConfiguration.DoesNotExist:
```

Comment:
`installation_id` could be `None` which may match the queryset if there are `installation_id` matching records with `None` in them. I would add an explicit check for `installation_id is None` and handle that case separately.

Reply:
Done